src/components/Square.js

```diff
@@ -1,19 +1,26 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import './Square.css'
+import './Square.css';
 
-const Square = (props) => {
-  // For Wave 1 enable this 
-  //  Component to alert a parent 
+const Square = ({ value, id, onClickCallback }) => {
+  const updateSquare = () => {
+    // console.log(id);
+    if (value === '') {
+      value = 'update';
+      onClickCallback(id);
+    }
```

Comment:
Good check to ensure squares don't get marked again by accident. It doesn't look like you're using the string `update` anywhere else in your code. I assume it was for testing purposes. I remove this line after cloning your project and looks like it didn't affect anything else. We can remove it!

```suggestion
const Square = ({ value, id, onClickCallback }) => {
  const updateSquare = () => {
    // console.log(id);
    if (value === '') {
      onClickCallback(id);
    }
```